Resources/public/js/views/actions/ez-translateactionview.js

```diff
@@ -62,15 +62,35 @@ YUI.add('ez-translateactionview', function (Y) {
 
                 location: this.get('location').toJSON(),
                 content: this.get('content').toJSON(),
-                translations: translationsList,
-                canDeleteTranslations: translationsList.length > 1,
+                translations: this._decorateTranslationsList(translationsList),
                 firstLanguagesCode: firstLanguageCodes,
                 moreTranslationCount: moreTranslationCount
             }));
 
             return this;
         },
 
+        /**
+         * decorates the list of language codes by adding true/false flag which
+         * determines whether delete button should be visible
+         *
+         * @method _decorateTranslationsList
+         * @param {Array} translationsList list of translation that will be decorated
+         * @return {Object} object which contains language codes with visibility flags
+         * @protected
+         */
+        _decorateTranslationsList: function (translationsList) {
```

Comment:
Maybe `_flagRemovableTranslations`
Also `translations` instead of `translationsList`. It's nicer to read :)